src/main/scala/group/scala/karazin/mongo4s/model.scala

```diff
@@ -192,10 +192,11 @@ object model:
 
     object CollStats:
       final case class LatencyStats(histograms: Boolean)
+      final case class StorageStats(scale: Int)
```

Comment:
MongoDB also supports non-integer scale factor (e.g. float). But I'm not sure if this is critical and whether it needs to be fixed.

Reply:
Yeas, it does, but it only takes the integer part of the number anyway. Good point though.